### PR TITLE
added documentation for adding facts 

### PR DESCRIPTION
**Pull request by seijisakurai**

### CODE REVIEW

File: docs/operator/advanced/api.md [hunk 1]
```diff
@@ -102,6 +102,14 @@ curl -X PUT -sk -H $TOKEN -H 'Content-Type: application/json' "https://localhost
 }' | json_pp
 ```
 
+#### Add multiple facts to an agent
+
+You can add multiple facts to an agent by passing facts
+
+```bash
+curl -X POST -sk -H 'Authorization:blue' "https://localhost:8888/v1/agents/privateducky/facts" -d '[{"key":"hello", "value":"world", "scope":"agent"},{"key":"fourth"}]' -H 'Content-Type: application/json'
```
Comment (privateducky) [inline docs/operator/advanced/api.md:110]: describe that "privateducky" is the agent in this case.

also make the Authorization header consistent across all API examples in this doc.

Comment (seijisakurai) [reply docs/operator/advanced/api.md]: done

File: docs/operator/advanced/api.md [hunk 2]
```diff
@@ -104,10 +104,10 @@ curl -X PUT -sk -H $TOKEN -H 'Content-Type: application/json' "https://localhost
 
 #### Add multiple facts to an agent
 
-You can add multiple facts to an agent by passing facts
+You can add multiple facts to an agent by passing facts. Replace agent_name with the name of the agent.
 
 ```bash
-curl -X POST -sk -H 'Authorization:blue' "https://localhost:8888/v1/agents/privateducky/facts" -d '[{"key":"hello", "value":"world", "scope":"agent"},{"key":"fourth"}]' -H 'Content-Type: application/json'
+curl -X POST -sk -H $TOKEN "https://localhost:8888/v1/agents/agent_name/facts" -d '[{"key":"hello", "value":"world", "scope":"agent"},{"key":"fourth"}]' -H 'Content-Type: application/json'
```
Comment (privateducky) [inline docs/operator/advanced/api.md:110]: good. now you need to replace all instances of $TOKEN in this file with "Authorization: $TOKEN"

Comment (seijisakurai) [reply docs/operator/advanced/api.md:110]: In the document it is mentioned that users need to replace $TOKEN with "Authorization: YOURTOKEN". Should we still replace them?

Comment (privateducky) [reply docs/operator/advanced/api.md:110]: I think I'd say something like: you must also pass in an authentication header like this... 

Then you can remove $token from all examples

Comment (seijisakurai) [reply docs/operator/advanced/api.md]: Went ahead with replacing the instances with "Authorization: YOUR_API_TOKEN".

**Review by privateducky — CHANGES_REQUESTED** [review]
Haha almost!

Going this direction, change YOUR_TOKEN back to $TOKEN so people can set an env variable then just copy this command.

**Review by privateducky — APPROVED** [review]

**Pull request merged**